src/main/frontend/Admin.html

```diff
@@ -166,26 +200,33 @@ <h1>Flights Database</h1>
                 // Handling click for the header row
                 row.insertCell(-1).innerHTML = '';
                 row.firstChild.id = "buttons";
-                row.onclick = function (event) {
-                    if (event.target.tagName === "TD" && event.target.id !== "buttons") {
-                        if (cellContent === event.target.innerHTML) {
+                for (let j = 0; j < lines[i].length; j++) {
+                    // Add arrow indicators for sorting order next to each header cell
+                    const headerCell = row.insertCell(-1);
+                    headerCell.innerHTML = lines[i][j];
+                    if (lines[i][j] === cellContent) {
+                        headerCell.innerHTML += sortOrder === 'asc' ? '▼' : '▲';
```

Comment:
This is one way to do it. If you could also make it an imgage  but that fully up to you